tests/test-driver-decode-buferr.c

```diff
@@ -1,4 +1,4 @@
-//!compile = as -c -o {ofile} {infile} && {cc} {ccflags} -o {outfile} {ofile} {driver} ../libdbrew.a -I../include
+//!compile = as -c -o {ofile} {infile} && {cc} {ccflags} -o {outfile} {ofile} {driver}  ../build/src/libdbrew-test.a -I../include
```

Comment:
This comment is only needed for Makefiles/test.py. The test script only supports Makefiles/in-tree builds and it doesn't make sense to adopt this for meson. Hard-coding the build path in out-of-tree build doesn't seem to be a good idea.